gcp-gke-core-services/_artifacts.tf

```diff
@@ -0,0 +1,40 @@
+
```

Comment:
copy-pasta

Reply:
why does our core services module make the artifact instead of the parent bundle? I think as we move more towards this module pattern bundles become more wiring and data munging making these modules really useful outside of bundles.

Reply:
Yeah it definitely felt a little odd. I'll yank it and keep the artifacts in the bundles 👍